galsim/correlatednoise.py

```diff
@@ -1465,3 +1506,86 @@ def __repr__(self):
 
     def __str__(self):
         return "galsim.UncorrelatedNoise(variance=%r, wcs=%s)"%(self.variance, self.wcs)
+
+
+class CovarianceSpectrum(object):
+    """Class to hold a `ChromaticSum` noise covariance spectrum (which is a generalization of a
+    power spectrum or equivalently a correlation function).
+
+    Analogous to how a galsim.CorrelatedNoise object stores the variance and covariance of a
+    galsim.Image object, a galsim.CovarianceSpectrum stores the variance and covariance of the
+    Fourier mode amplitudes in different components of a `ChromaticSum`.  Note that the covariance
+    in question exists between different SED-components of the `ChromaticSum`, and not between
+    different Fourier modes, which are assumed to be uncorrelated.  This structure arises naturally
+    for `RealChromaticGalaxy`s (see devel/modules/CGNotes.pdf for more details).
```

Comment:
tbh I can never remember which one of these I landed on.  Fixed here and elsewhere.